definitions/ext-rf_scanner/definition.yml

```diff
@@ -0,0 +1,51 @@
+#  [mandatory] The domain of an entity, default to EXT unless otherwise indicated by the maintainers.
+domain: EXT
+# [mandatory] The type of an entity. Must be unique within its domain, but may be duplicated between domains.
+# Please add a brief description of what your type is here on the definition file.
+type: RF-Scanner
```

Comment:
The PR is still failing because this should be uppercase and it cannot contain dashes, so maybe it can be `RF_SCANNER` instead